File: android-v0-compose/app/src/main/java/com/example/v0clone/data/model/AppVersionInfo.kt
package com.xlwl.AiMian.data.model

data class AppVersionInfo(
    val id: String,
    val platform: String,
    val versionName: String,
    val versionCode: Int,
    val downloadUrl: String,
    val releaseNotes: String?,
    val isMandatory: Boolean,
    val isActive: Boolean,
    val shouldUpdate: Boolean = false,
    val forceUpdate: Boolean = false
)

File: iOS-v0/Sources/Models/AppVersionInfo.swift
import Foundation

struct AppVersionInfo: Decodable, Sendable {
  let id: String
  let platform: String
  let versionName: String
  let versionCode: Int
  let downloadUrl: String
  let releaseNotes: String?
  let isMandatory: Bool
  let isActive: Bool
  let shouldUpdate: Bool
  let forceUpdate: Bool
}
